taker/src/routes.rs

```diff
@@ -64,8 +67,12 @@ pub async fn feed(
     rx_maker_status: &State<watch::Receiver<ConnectionStatus>>,
     rx_maker_identity: &State<watch::Receiver<Option<identify::PeerInfo>>>,
     identity_info: &State<IdentityInfo>,
-    _auth: Authenticated,
+    user: User,
 ) -> EventStream![] {
+    if user.expired {
+        // TODO fail this call. But how?
```

Comment:
🤔 What should the behavior of this fail be?

I don't think we want to blow up the `EventStream`, but we would like to report back to the user that there is a problem. Given that the daemon is pushing information to the consumer (the UI), to solve this we would have to introduce an error stream. Or we change constraints around the API; https://github.com/itchysats/itchysats/issues/2906 might help but would not solve this problem completely.

Reply:
❓ The current behavior would be that the user still receives information even if the cookie is already expired, correct?
Given that the user then cannot trigger any actions (because those endpoints should be guarded) that is acceptable.

Reply:
> 🤔 What should the behavior of this fail be?

I think that this function returns mission critical information and should require authentication. With http best practices in mind, ideally this function returns `HTTP::unauthorized` or `forbidden` but I was not able to return this http error type here.

Reply:
> Generally I would also not go for an auth cookie thats valid for 7 days (I know its different here because if we run frontend and backend in the home network - however we are only assuming that the deployment is like that)

What expiry would you go for? I'm happy to change it.

Reply:
@bonomat typically you go for a few minutes to reduce the risk of accepting a leaked token / cookie. To keep the user session alive you can use silent authentications or refresh tokens..

Typically I would go for an auth2.0 authorization code flow with PKCE in that case. That would also allow us to connect to other identity providers if needed in the future. (we could even allow the user to use a social login)

Reply:
> @bonomat typically you go for a few minutes to reduce the risk of accepting a leaked token / cookie. To keep the user session alive you can use silent authentications or refresh tokens..

Interesting, I'll create a follow-up ticket on this once this is merged.